code/game/objects/machinery/doors/airlock_types.dm

```diff
@@ -550,7 +550,7 @@
 	icon = 'icons/obj/doors/mainship/medidoor.dmi'
 	req_access = list(ACCESS_MARINE_RESEARCH)
 
-/obj/machinery/door/airlock/mainship/research
+/obj/machinery/door/airlock/mainship/rebel/research
```

Comment:
Put /rebel/research as opposed to /research/rebel in order to maintain uniformity, idk if that will impact anything else.